profile/README.md

```diff
@@ -0,0 +1,34 @@
+# Welcome to Keploy 👋
+
+<p style="text-align:center;" align="center">
+      <td align="center"><img src="https://avatars.githubusercontent.com/u/92252339?s=200&v=4" width="200px;" alt=""/></a> 
+      </td>
+</p>
+
+<h3 align="center">
+  <b><a href="https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA">Slack</a></b>
+  •
+  <a href="https://github.com/keploy">Github</a>
+  •
+  <a href="https://www.linkedin.com/company/keploy/">LinkedIN</a>
+  •
+  <a href="https://twitter.com/Keployio">Twitter</a>
+  •
+  <a href="https://www.youtube.com/channel/UC6OTg7F4o0WkmNtSoob34lg">Youtube</a>
+  
+</h3>
+
+[Keyploy](https://keploy.io/) is a no-code testing platform that generates tests from API calls. It converts API calls into testcases. Mocks are automatically generated with the actual request/responses.
+
+### 🏂 Haven't tried Keyploy yet? Below is a list of the most **important repositories**
+
+<br>
+
+- [Keyploy](https://github.com/keploy/keploy) : No code API testing platform. Create unit tests and data mocks from API calls.
+- [go-sdk](https://github.com/keploy/go-sdk) : Client SDK for Keploy API Platform. No Code API testing using production traffic.
+- [UI](https://github.com/keploy/ui) : UI for Keploy platform that does API testing by dependency mocking without writing any test cases.
+- [example-url-shortener](https://github.com/keploy/example-url-shortener) : Sample application to understand keploy integration.
```

Comment:
We have archived this repo and collated all samples in one repo : https://github.com/keploy/samples-go